package.json

```diff
@@ -82,5 +82,8 @@
     "propagators/*",
     "detectors/node/*",
     "metapackages/*"
-  ]
+  ],
+  "dependencies": {
+    "react": "^18.0.0"
```

Comment:
should this one be added in the package folder instead? Is it more of a peer dependency as well?

Reply:
yep, good catch! probably it was my hitting `npm install react` in the wrong place. I will fix it

Reply:
fixed.